codegen/tests/dbendpoints/servicehandler.go

```diff
@@ -34,10 +35,16 @@ func NewServiceHandler(genCallback GenCallback, serviceInterface *ServiceInterfa
 	return &ServiceHandler{genCallback, serviceInterface, db}
 }
 
+// Handler Error
+func (s *ServiceHandler) HandleError(ctx context.Context, w http.ResponseWriter, kind common.Kind, message string, cause error) {
```

Comment:
`HandleError` can be private: `handleError`

Reply:
Thanks Andrew, just make this lowercase will make it private?

Reply:
Yes.